examples/sample-pub/views/article.html

```diff
@@ -110,6 +110,7 @@ <h1 class="heading"><% article.title %></h1>
           <p class="byline">By <% article.author %> on <% article.date %></p>
           <div class="hr"></div>
 
+          <div id="swgId" style="display:none;height:10px;min-width: 20px;background-color: #ff1"></div>
```

Comment:
Publisher would require to specify the container `DIV`, where `smartbutton` would be injected by the API call. 

Need to finalize, what `css` `style` attributes are allowed/accepted by SwG.